src/domain/entities/TranslatableText.ts

```diff
@@ -16,3 +17,24 @@ export const buildTranslate = (locale: string): TranslateMethod => {
 };
 
 export type TranslateMethod = (string: TranslatableText) => string;
+
+export const importTranslate = <T extends TranslatableText>(item: T, language: string, term: string | undefined): T => {
```

Comment:
Use `Maybe` for `undefined | T`. But in this repo we have declared `Maybe` as `type Maybe<T> = T | undefined | null`; so I'm not sure about this (@tokland)

Reply:
99% of the time is not a problem to have null also in the maybe, if it types, it's ok.